Author: Okinea Dev <hi@okinea.dev>
Description: This patch unlocks the anti-rollback protection by setting the anti_version to 0.

diff --git a/images/anti_version.txt b/images/anti_version.txt
index 0000000..0000000 100644
--- a/images/anti_version.txt
+++ b/images/anti_version.txt
@@ -1 +1 @@
-1
+0
